instagram-build/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import CommentSection from '../CommentSection/CommentSection'
+import "./Post.css"
+
+function PostContainer(props) {
+    return (
+        <div className="container">
+            <div className="posts">
+            {props.dummy.map(dummy => (
```

Comment:
we actually want to map in app.js and make a post container for each post (hence the whole reusable component idea), slight refactor would get you there